backend/src/main/resources/application.properties

```
@@ -0,0 +1,6 @@
+# H2
+spring.datasource.url=jdbc:h2:file:C:/temp/test
```

```suggestion
spring.datasource.url=jdbc:h2:file:C:/temp/test
```

Make this a relative path not an absolute one - as this won't be working on Linux/Unix machines